arbitrum-docs/how-arbitrum-works/01-a-gentle-introduction.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: 'A gentle introduction'
+description: 'Learn the fundamentals of Nitro, Arbitrum stack.'
+author: dzgoldman
+sme: dzgoldman
+user_story: As a current or prospective Arbitrum user, I need learn more about Nitros design.
+content_type: get-started
+---
+
+import ImageWithCaption from '@site/src/components/ImageCaptions/';
+
+This document is a deep-dive explanation of Arbitrum Nitro’s design and the rationale for it. This isn’t API documentation, nor is it a guided tour of the code--look elsewhere for those. “Inside Arbitrum Nitro” is for people who want to understand Nitro's design.
+
+The body of this document will describe Arbitrum Rollup, the primary use case of the Nitro technology and the one used on the Arbitrum One chain. There is a variant use case, called AnyTrust, which is used by the Arbitrum Nova chain. AnyTrust is covered by a section at the end of this document.
+
+## Why use Arbitrum? Why use Nitro?
+
+Arbitrum is an L2 scaling solution for Ethereum, offering a unique combination of benefits:
+
+- Trustless security: security rooted in Ethereum, with any one party able to ensure correct Layer 2 results
+- Compatibility with Ethereum: able to run unmodified EVM contracts and unmodified Ethereum transactions
+- Scalability: moving contracts’ computation and storage off of the main Ethereum chain, allowing much higher throughput
+- Minimum cost: designed and engineered to minimize the L1 gas footprint of the system, minimizing per-transaction cost.
+
+Some other Layer 2 systems provide some of these features, but to our knowledge no other system offers the same combination of features at the same cost.
+
+Nitro is a major upgrade to Arbitrum, improving over "classic" Arbitrum in several ways:
```

Comment:
We could rephrase this to remove the mention to "Classic". Something like "Some of the features included in Arbitrum Nitro are:" (probably changing "features" to something different, since it's used a lot in the previous paragraph.

Reply:
Will rephrase